resources/views/courses/create.blade.php

```diff
@@ -11,36 +11,39 @@
 		<!-- Course name -->
 		<div class="field">
 			<label for="courseName">Course name</label>
-			<input type="text" name="courseName" id="courseName" required>
+			<input type="text" name="courseName" id="courseName">
 		</div>
 		<!-- Teacher(s) -->
-		<div class="field" title="Are you giving this course with another teacher?">
+		<div class="field">
 			<label for="teachers">Teacher(s)</label>
-			<select class="ui fluid search dropdown" name="teachers[]" id="teachers" multiple required>
+			<select class="ui fluid search dropdown" name="teachers[]" id="teachers" multiple>
 				@foreach ($teachers as $teacher)
-					<option value={{ $teacher['id'] }}> {{ $teacher['name'] }} </option>
+					<option value={{ $teacher['id'] }} {{ ($teacher['id'] == Auth::user()->id) ? 'selected' : '' }}> {{ $teacher['name'] }} </option>
 				@endforeach
 			</select>
+			<legend>Are you giving this course with another teacher?</legend>
 		</div>
 		<!-- Course semester -->
 		<div class="field">
 			<label for="courseSemester">Course semester</label>
-			<input type="text" name="courseSemester" id="courseSemester" placeholder="Spring/Summer/Fall/Winter and year.  Ex. Fall 2018" title="When is this course taking place?"required>
+			<input type="text" name="courseSemester" id="courseSemester">
+			<legend>When is this course taking place? Spring/Summer/Fall/Winter and year. Ex. Fall 2018</legend>
```

Comment:
Nit-pick: "e.g. Fall 2018", per the style guide.

Reply:
Let's not use the `legend` attribute ([as this is meant for another thing](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend)). You can use a `p` here.